packages/shared/src/GeolocationContext.js

```diff
@@ -61,3 +103,22 @@ class Provider extends React.Component<Props, State> {
 }
 
 export default { Consumer, Provider };
+
+export const withGeolocationContext = (Component: React.ElementType) => {
+  const withGeolocationContext = (props: {}) => {
+    return (
+      <Provider shouldNotUpdateGeolocationComponentDidMount={true}>
```

Comment:
It is better if this provider is just defined once application wide. Please rather add it to `MMB/index.js`

Also not sure about this prop. Maybe we should just always fail silently instead, or pass a callback to `updateGeolocation` so the consumer can decide what to do with the error.